src/Curiosity/curiosity.tsx

```diff
@@ -5,25 +5,48 @@ import { RoverPhoto } from '../RoverPhoto/RoverPhoto';
 
```

Comment:
I forgot to say it on the last review but we should add some tests for the roverPhoto and roverTopPhoto components, the roverTopPhoto component can also contain a test for it's props that are passed in similar to the tests Chloe and Hina added in RoverSummary.test.tsx